### ISSUE TITLE
Misleading url mentioned for testing the visualizing component

### ISSUE THREAD

**Issue by jessica-jordan** [biojs/edu#52]
In the beginner's tutorial there is a misleading mention of the /snippets url in the tutorial that the user shall check to implement and test their viz-component. The url doesn't reflect our current folder structure in a biojs package development and it seems to be just outdated and replaced by the /examples url instead.

Please fix all instances where a user is asked to go to the url /snippets with the actual url /examples.

**Comment by wilzbach** [comment]
Yep it was previously "snippets", but we renamed it a couple of months ago ;-)

**Comment by jessica-jordan** [comment]
Yes, just saw that one on the edu pages today as one student who got started on BioJS stumbled upon this. I'll check on that once I find time later.
